docs/developer/data_import.md

```diff
@@ -29,12 +29,12 @@ The data import app selection is configured in [`staging_upload.json`](../../kba
 
 Each `app_info` object has the following accepted keys:
 * `app_id`: the app to use for importing the file in the format `module_name/app_name` (e.g. `kb_uploadmethods/import_genbank_as_genome_from_staging`). This is the only required field.
-* `app_input_param`: this is the parameter for setting the file name in the app
-* `app_input_param_type`: describes what the `app_input_param` is expected to be. Accepts either `string` or `list` (list of strings)
+* `app_input_param`: this is the parameter for setting the file name in the app.
```

Comment:
Can you update as per https://github.com/kbase/narrative/pull/2605#discussion_r768944647

Reply:
I commented on that PR, but I'll just copy it here - that doesn't "override" the spec, because there isn't a specific "THIS IS THE INPUT FILE" field there. It is, by the nature of the app specs, possibly ambiguous.

Reply:
Okay, how about adding some text along those lines saying that the app doesn't provide an unambiguous input file marker hence the need for this field?

Reply:
Ok, done.